policy/release/attestation_task_bundle.rego

```diff
@@ -63,7 +63,8 @@ warn[result] {
 #   is the most recent acceptable one. See the list of acceptable
 #   task bundles at xref:acceptable_bundles.adoc#_task_bundles[Acceptable Bundles] or look at
 #   link:https://github.com/hacbs-contract/ec-policies/blob/main/data/acceptable_tekton_bundles.yml[data/acceptable_tekton_bundles.yml]
-#   in this git repository.
+#   in this git repository. The meaning of an acceptable bundle is explained in
+#   xref:acceptable_bundles.adoc#_task_bundles[Acceptable Bundles]
```

Comment:
@lcarva Do you mean to update the meaning of acceptable bundle for line 84 of task title `Task bundle is not acceptable`?

Reply:
https://github.com/hacbs-contract/ec-policies/pull/167